airflow/utils/serve_logs.py

```diff
@@ -73,10 +74,47 @@ def serve_logs_view(filename):
     return flask_app
 
 
+class StandaloneGunicornApplication(gunicorn.app.base.BaseApplication):
+    """
+    Standalone Gunicorn application/serve for usage with any WSGI-application.
+
+    Code inspired by an example from the Gunicorn documentation.
+    https://github.com/benoitc/gunicorn/blob/cf55d2cec277f220ebd605989ce78ad1bb553c46/examples/standalone_app.py
+
+    For details, about standalone gunicorn application, see:
+    https://docs.gunicorn.org/en/stable/custom.html
+    """
+
+    def __init__(self, app, options=None):
+        self.options = options or {}
+        self.application = app
+        super().__init__()
+
+    def load_config(self):
+        config = {
+            key: value
+            for key, value in self.options.items()
+            if key in self.cfg.settings and value is not None
+        }
+        for key, value in config.items():
```

Comment:
```suggestion
        for key, value in self.options.items():
```

I'm not sure we need this? Seems odd to silently drop options that are passed, same with None.

Reply:
@mik-laj, was this intentionally resolved without a comment?

Reply:
I did something wrong. I accepted this change, but I don't know why I can't see it now.

Reply:
Ah, weird 🍺

Reply:
I probably use the `--force` too much.